src/policy/semantic.rs

```diff
@@ -380,7 +380,7 @@ impl_from_tree!(
                 // thresh(1) and thresh(n) are disallowed in semantic policies
                 if thresh <= 1 || thresh >= (nsubs as u32 - 1) {
                     return Err(errstr(
-                        "Semantic Policy thresh cannot have k = 1 or k =n, use `and`/`or` instead",
+                        "Semantic Policy thresh cannot have k=1 or k==n, use `and`/`or` instead",
```

Comment:
In 939695dfcb6508e0062a50b80e5e68dcfa5b0e4b:

I think single equality was fine, but I don't feel strongly about it. But you need to the same = operator in both cases :).

Reply:
lol, its hard to get good help. Just added the space into the original line. I've been chided recently about doing trivial patches and slowing the dev process down, this is definitely a case of that :(

cc @kiminuo - woops.